tabular/src/autogluon/tabular/models/catboost/callbacks.py

```diff
@@ -139,10 +142,10 @@ def __init__(self, stopping_rounds, eval_metric, compare_key='validation'):
 
     def after_iteration(self, info):
         is_best_iter = False
-        if self.eval_metric_name.startswith('MultiQuantile'):
+        if self.eval_metric_name.startswith(CATBOOST_QUANTILE_PREFIX):
```

Comment:
while this is a tiny overhead, might as well just have this boolean stored as a `self.is_quantile` to avoid having to do this every iteration.